demisto_sdk/commands/validate/tests/IF_validators_test.py

```diff
@@ -0,0 +1,332 @@
+        create_incident_field_object(["cliName"], ["parent"])
+    ]
+
+    results = IsCliNameReservedWordValidator().is_valid(content_items)
+    assert (
+        results[0].message
+        == "`cliName` field can not be `parent` as it's a builtin key."
+    )
+
+
+def test_IsValidContentFieldValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a content value True
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [
+        create_incident_field_object(["content"], [True])
+    ]
+
+    results = IsValidContentFieldValidator().is_valid(content_items)
+    assert not results
+
+
+def test_IsValidSystemFlagValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a system value False
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [
+        create_incident_field_object(["system"], [False])
+    ]
+
+    results = IsValidSystemFlagValidator().is_valid(content_items)
+    assert not results
+
+
+def test_IsValidFieldTypeValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a valid type value
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [
+        create_incident_field_object(["type"], ["html"])
+    ]
+
+    results = IsValidFieldTypeValidator().is_valid(content_items)
+    assert not results
+
+
+def test_IsValidGroupFieldValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a group value 0
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [create_incident_field_object(["group"], [0])]
+
+    results = IsValidGroupFieldValidator().is_valid(content_items)
+    assert not results
+
+
+def test_IsCliNameFieldAlphanumericValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a cliName value
+          that is alphanumeric and lowercase letters
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [
+        create_incident_field_object(["cliName"], ["test1234"])
+    ]
+
+    results = IsCliNameFieldAlphanumericValidator().is_valid(content_items)
+    assert not results
+
+
+def test_IsCliNameReservedWordValidator_valid():
+    """
+    Given:
+        - IncidentField content items with a cliName value that is not reserve word
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that no ValidationResult returned
+    """
+    content_items: List[IncidentField] = [
+        create_incident_field_object(["cliName"], ["test"])
```

Comment:
```suggestion
        create_incident_field_object(["cliName"], ["foo"])
```
("test" sounds like it _could_ be a reserved word)